src/cmd/cmd.go

```diff
@@ -299,7 +299,15 @@ func CheckUpgrade(version string) {
 	if latestTag == version {
 		utils.PrintInfo("Spicetify up-to-date")
 	} else {
-		utils.PrintWarning("New version available!")
-		utils.PrintWarning(`Run "spicetify upgrade" or using package manager to upgrade spicetify`)
+		newVersionMsg := "New version available: v" + latestTag + " (current is: v" + version + ")"
+		howToUpgradeMsg := `Run "spicetify upgrade" or using package manager to upgrade spicetify`
+
+		if printLevel == "info" {
+			utils.PrintInfo(newVersionMsg)
+			utils.PrintInfo(howToUpgradeMsg)
+		} else if printLevel == "warning" {
+			utils.PrintWarning(newVersionMsg)
+			utils.PrintWarning(howToUpgradeMsg)
+		}
```

Comment:
Makes more sense to have different printLevels depending on the context
(it shouldn't be "warning" when running `spicetify --update=check`